### ISSUE TITLE
BUG: "dnu commands install" fails to find .sha512

### ISSUE THREAD

**Issue by giggio** [aspnet/dnx#2192]
I have a project.json that depends on `FluentMigrator` version `1.5.1.0`. This package defines its version as `1.5.1.0`. If I try to install my package (FluentMigrator.Runner.Aspnet):

````
dnu commands install FluentMigrator.Runner.Aspnet --no-cache
````

It fails because it can't find the `.sha512` file:

````
Could not find a part of the path 'C:\Users\username\.dnx\bin\packages\FluentMigrator\1.5.1.0\FluentMigrator.1.5.1.0.nupkg.sha512'.
````

It is looking for the `1.5.1.0` directory, but `dnu` is saving it without the last `.0`, so I have on disk only the `$HOME\.dnx\bin\packages\FluentMigrator\1.5.1` directory. And installation fails.

**Comment by giggio** [comment]
The problem also happens on `dnu restore` for the package.

**Comment by davidfowl** [comment]
The problem is that these packages have inconsistent nuspec versions. 

@muratg @ChengTian @anurse  we should fix this ASAP. A bunch of packages are running into this. We need to resolve the inconsistencies between package version in the nuspec and elsewhere.

**Comment by muratg** [comment]
:+1:

**Comment by troydai** [comment]
@giggio Could you let us know what's version of the DNX you're using?

I fail to repo this issue with either beta5 or beta6 latest with this command (targeting alpha3 since you work around the problem in alpha4):
```
dnu commands install FluentMigrator.Runner.AspNet 1.0.0-alpha3 --no-cache
```

**Comment by davidfowl** [comment]
This absolutely repros. I can repro it myself without a commands install.

**Comment by giggio** [comment]
@troydai that is because I have forked the original FluentMigrator.Runner and uploaded a correct version to Nuget (which I plan to remove as soon as the original is fixed).

**Comment by troydai** [comment]
@giggio I can repo the issue now. Working on it.

**Comment by troydai** [comment]
Fixed in cae8aab5b38e0039f03eb87a8d994e3035179206

**Comment by giggio** [comment]
@davidfowl @troydai Any workarounds at this point to work with these packages and `beta5`?
